app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -383,7 +383,7 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope(), V
                 addDnsServer("10.0.0.1")
                 // just so that we can connect to our BE
                 // TODO should we protect all comms with our controller BE? other VPNs do that
-                safelyAddDisallowedApps(listOf("com.duckduckgo.mobile.android", "com.duckduckgo.mobile.android.debug"))
+                safelyAddDisallowedApps(listOf("com.duckduckgo.mobile.android", "com.duckduckgo.mobile.android.debug", "com.duckduckgo.dev.android"))
```

Comment:
Should we change this to the following?
```suggestion
                safelyAddDisallowedApps(listOf(this@TrackerBlockingVpnService.packageName))
```